src/app/streamlit/main.py

```
@@ -7,10 +7,10 @@
 import yaml
 from PIL import Image
 
-from app.api.dependencies import get_current_user
-from app.core.logger import logging
-from app.schemas.agent import Agent
-from app.streamlit.streamlit_auth import (
+from src.app.api.dependencies import get_current_user
```

Ruff

`src.app.api.dependencies.get_current_user` imported but unused

Remove unused import: `src.app.api.dependencies.get_current_user`

(F401)